activate.sh

```diff
@@ -28,15 +29,6 @@ EOF
 echo "master: localhost" > /etc/salt/minion.d/minion.conf
```

Comment:
Why don't we surround all this code with a

```bash
if [ ! -f /etc/sysconfig/caasp-activation ] ; then
   touch /etc/sysconfig/caasp-activation
   # the rest of the code here...
fi
```

so we are completely sure this is run only once after the installation?